src/PanelTraits/Create.php

```diff
@@ -22,12 +25,12 @@ public function create($data)
         $data = $this->decodeJsonCastedAttributes($data, 'create');
         $data = $this->compactFakeFields($data, 'create');
 
-        // ommit the n-n relationships when updating the eloquent item
+        // omit the n-n relationships when updating the eloquent item
         $nn_relationships = array_pluck($this->getRelationFieldsWithPivot('create'), 'name');
         $item = $this->model->create(array_except($data, $nn_relationships));
 
         // if there are any relationships available, also sync those
-        $this->syncPivot($item, $data);
+        $this->createRelations($item, $data);
```

Comment:
What I meant by not exposing it in the public API was to restrict its visibility so a user could not call `$this->crud->syncPivot($model, $data, $form)` directly in a crud controller, nor to override it there.

The `syncPivot` method should still exist, but should be protected. This way, it can still be overridden or exposed as a public method, if that's the case, by extending the `CrudPanel` class. 

There are still things to do until we get there, like letting the user inject a custom 'CrudPanel' class in the `CrudController`. Deprecating it is just a first step.